benchmark/run_darkhorse.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# usage: run DarkHorse software
+database_fp=$1
+diamond_nr=$2
+diamond_tabular_query=$3
+darkhorse_config_file=$4
+darkhose_install_dir=$5
+query_species_coding_seqs_fp=$6
+working_dir=$7
+threads=$8
+
+mkdir -p "${working_dir}/diamond"
+
+## Align with DIAMOND if alignments don't exist
+if [ "${diamond_tabular_query}" == "None" ]
```

Comment:
Just out of curiosity, you don't need the "" in "${diamond_tabular_query}", right?